spec.html

```diff
@@ -313,9 +313,7 @@
       <h1>BigInt::leftShift (_x_, _y_)</h1>
       <p>The abstract operation BigInt::leftShift with two arguments _x_ and _y_ of BigInt:</p>
       <emu-alg>
-        1. If _y_ &lt; 0,
-          1. Return a BigInt representing _x_ divided by 2<sup>_y_</sup>, rounding down to the nearest integer, including for negative numbers.
```

Comment:
Okay, I need to read this when I have more time, but to answer your question: yes.